### ISSUE TITLE
assert len(indices) == len(set(indices)) causes problems

### ISSUE THREAD

**Issue by eddie-dunn** [hcpl/xkbgroup#2]
On line 496 in core.py the assertion `assert len(indices) == len(set(indices))` is made. This means I cannot run many of the methods on xkbgroup.XKeyboard instances.

E.g.:
```ipython
In [32]: xkb = xkbgroup.XKeyboard()

In [33]: xkb.groups_data
---------------------------------------------------------------------------
AssertionError                            Traceback (most recent call last)
<ipython-input-33-b29f1ff8f494> in <module>()
----> 1 xkb.groups_data

/home/eddie/.virtualenvs/keyboard_switcher/lib/python3.6/site-packages/xkbgroup/core.py in groups_data(self)
    231                           in zip(range(self.groups_count),
    232                                  self.groups_names,
--> 233                                  self.groups_symbols,
    234                                  self.groups_variants))
    235 

/home/eddie/.virtualenvs/keyboard_switcher/lib/python3.6/site-packages/xkbgroup/core.py in groups_symbols(self)
    269         :type: list of str
    270         """
--> 271         return _ListProxy(symdata.symbol for symdata in self._symboldata_list)
    272 
    273     @property

/home/eddie/.virtualenvs/keyboard_switcher/lib/python3.6/site-packages/xkbgroup/core.py in _symboldata_list(self)
    436         if symbol_str_atom != None_:
    437             b_symbol_str = XGetAtomName(self._display, symbol_str_atom)
--> 438             return _parse_symbols(b_symbol_str.decode(), self.non_symbols)
    439         else:
    440             raise X11Error("Failed to get symbol names.")

/home/eddie/.virtualenvs/keyboard_switcher/lib/python3.6/site-packages/xkbgroup/core.py in _parse_symbols(symbols_str, non_symbols, default_index)
    494 
    495     indices = [symdata.index for symdata in symboldata_list]
--> 496     assert len(indices) == len(set(indices))    # No doubles
    497 
    498     return symboldata_list

AssertionError:
```

What is the reason for asserting against doubles? Wouldn't it be better to just filter them out?

My setup:
```
$ setxkbmap -query -v 10
Setting verbose level to 10
locale is C
Trying to load rules file ./rules/evdev...
Trying to load rules file /usr/share/X11/xkb/rules/evdev...
Success.
Applied rules from evdev:
rules:      evdev
model:      pc105
layout:     se,us
options:    caps:escape,grp:shifts_toggle,grp_led:caps
Trying to build keymap using the following components:
keycodes:   evdev+aliases(qwerty)
types:      complete
compat:     complete+ledcaps(group_lock)
symbols:    pc+se+us:2+inet(evdev)+group(shifts_toggle)+capslock(escape)
geometry:   pc(pc105)
rules:      evdev
model:      pc105
layout:     se,us
options:    caps:escape,grp:shifts_toggle,grp_led:caps
```

**Comment by eddie-dunn** [comment]
solved by #4

**Comment by eddie-dunn** [comment]
Fixed merged in bb369b5d1777b217f983522cf590609bd9416c23
